website/docs/streaming-lakehouse/integrate-data-lakes/paimon.md

```diff
@@ -92,14 +96,29 @@ For further information, refer to Paimon’s [SQL Query documentation](https://p
 
 #### Union Read of Data in Fluss and Paimon
 
+##### Prerequisites
+Download the [fluss-lake-paimon-$FLUSS_VERSION$.jar](https://repo1.maven.org/maven2/org/apache/fluss/fluss-lake-paimon/$FLUSS_VERSION$/fluss-lake-paimon-$FLUSS_VERSION$.jar), and place it into `${FLINK_HOME}/lib`.
+
```

Comment:
Just to confirm: We'll deploy these JARs after releasing version 0.8.0, right? (Reference link: https://repo1.maven.org/maven2/org/apache/fluss/) Also, why aren't we using this link instead: https://github.com/apache/fluss/releases/download/v0.7.0/?

<img width="683" height="265" alt="Image" src="https://github.com/user-attachments/assets/4eeec2e3-23a2-4bb2-a152-f4812458d6b0" />

Reply:
Yes, we 'll deploy these jars. All jar download link is refer to `https://repo1.maven.org/maven2/org/apache/fluss/` in the doc